src/plugins/timelist/inspector/TimelistPropertiesView.vue

```diff
@@ -30,6 +30,23 @@
             These settings don't affect the view while editing, but will be applied after editing is
             finished.
           </div>
+          <div class="c-inspect-properties__label" title="Display Style">Display Style</div>
+          <div v-if="canEdit" class="c-inspect-properties__value">
+            <select v-model="isExpanded" @change="updateExpandedView()">
+              <option
+                v-for="(displayStyleOption, index) in displayStyleOptions"
+                :key="index"
+                :value="displayStyleOption.key"
+              >
+                {{ displayStyleOption.label }}
+              </option>
+            </select>
+          </div>
+          <div v-else class="c-inspect-properties__value">
+            {{ isExpanded ? displayStyleOptions[1].label : displayStyleOptions[0].label }}
```

Comment:
```suggestion
            {{ isExpanded ? 'Expanded' : 'Compact' }}
```

Reply:
I didn't want to use the hardcoded values here. I'll use a computed property instead.